src/main/java/com/google/devtools/build/lib/skyframe/BlacklistedPackagePrefixesFunction.java

```diff
@@ -97,7 +97,7 @@ public SkyValue compute(SkyKey key, Environment env)
 
     @Override
     public boolean processLine(String line) throws IOException {
-      if (!line.isEmpty()) {
+      if (!line.isEmpty() && !line.startsWith("#")) {
```

Comment:
Is it possible that a directory name starts with `#`?
`/` is surely disallowed in directory names, and absolute paths are disallowed (I hope) in the `.bazelignore`, so `//` as the comment prefix seems safer than `#`. WDYT?

Reply:
Right, I just noticed you test this in the shell test, but IMO the behavior is incorrect now.
So my question is still open :)

Reply:
Hm, I think my main motivation for using was to make this work the same way `gitignore` works. `#` seems to be the pseudo-standard for ignore files as far as I can tell. So I think user-expectation of a line `#some/path` is that it will be ignored.

Directory names starting with `#` seems very unlikely to me. But I get your point that this is technically breaking. I bet there are more `bazelignore` files where comments like `#some helpful text` "works" merely based on the fact that there is no directory with that name anyway.

Reply:
That's a good point about convention. You know what, that's actually a pretty compelling argument. Let's stay with `#`.